src/spec_procs.cpp

```diff
@@ -3985,8 +3985,8 @@ SPECIAL(quest_debug_scanner)
       snprintf(buf, sizeof(buf), "NPC %s's quest-related information:\r\n", GET_CHAR_NAME(to));
       snprintf(ENDOF(buf), sizeof(buf) - strlen(buf), "Overall max rep: %d, overall min rep: %d\r\n",
               get_johnson_overall_max_rep(to), get_johnson_overall_min_rep(to));
-      snprintf(ENDOF(buf), sizeof(buf) - strlen(buf), "SPARE1: %ld, SPARE2: %ld (corresponds to quest vnum %ld)\r\n",
-              GET_SPARE1(to), GET_SPARE2(to), GET_SPARE2(to) ? quest_table[GET_SPARE2(to)].vnum : -1);
+     //Remember to add here a list of all quests assigned to this johnson -- Nodens
```

Comment:
Looks like this todo didn't get completed yet, unless I missed it?

Reply:
No you're right. It's part of the second part (fixer mob/met johnsons tracking). Did not want to push a huge PR as it's harder to review. Obsolete display of current quest via `SPARE2` was removed from there so this is a todo comment tag I can search for that also serves to mark the change that is happening there.